docs/source/HISTORY.rst

```diff
@@ -1,6 +1,18 @@
 Release History
 ===============
 
+3.3.4 (2019-xx-xx)
+------------------
+
+- Fix receive command bugs related template labels
+- Fix receive command bug when importing to custom credential types
+- Fix receive command bug when host name was in multiple inventories
+- Fix bugs with special characters in config options
+- Fix bug using HipChat notification "notify" option
+- Password prompt written to stderr now
+- Support prompting for variables with workflows
```

Comment:
Its not something major, but shall we add something for #688 as well in this line ?
`Support prompting for variables and inventory with workflows ?`